packages/backend/src/app/user/user-service.ts

```diff
@@ -35,6 +35,19 @@ export const userService = {
 
         return userRepo.save(user)
     },
+    async verify({ userId }: { userId: string }): Promise<User> {
+        const user = await userRepo.findOneByOrFail({
+            id: userId,
+        })
+        if (user.status === UserStatus.SHADOW) {
+            await userRepo.update(userId, {
+                status: UserStatus.VERIFIED,
+            })
+        }
+        return userRepo.findOneByOrFail({
+            id: userId,
+        })
+    },
```

Comment:
I think that this method can be optimised to make less than 3 db calls to get the user verified, right?